src/sgame/sg_bot_public.h

```diff
@@ -0,0 +1,65 @@
+/*
+===========================================================================
+
+Unvanquished GPL Source Code
+Copyright (C) 1999-2005 Id Software, Inc.
+
+This file is part of the Unvanquished GPL Source Code (Unvanquished Source Code).
+
+Unvanquished is free software; you can redistribute it
+and/or modify it under the terms of the GNU General Public License as
+published by the Free Software Foundation; either version 2 of the License,
+or (at your option) any later version.
+
+Unvanquished is distributed in the hope that it will be
+useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Unvanquished; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+===========================================================================
+*/
+
+/*
+======================
+src/sgame/sg_bot.h
+
+This file contains the public interface used by sgame to interact with bots.
+======================
+*/
+
+#include "sg_struct.h"
+
+#ifndef __BOT_PUBLIC_HEADER
```

Comment:
Identifiers beginning with double underscore or underscore and capital letter are reserved for the compiler/stdlib

Reply:
I must say I've copied that from `sg_bot.h`

Reply:
Two wrongs don't make a right